src/core/sys/posix/aio.d

```diff
@@ -34,6 +34,66 @@ version (CRuntime_Glibc)
             off_t aio_offset;
             ubyte[32] __glibc_reserved;
         }
+
+    }
+    else version (FreeBSD)
+    {
+        struct __aiocb_private {
+            long status;
+            long error;
+            void *kernelinfo;
+        }
+
+        struct aiocb
+        {
+            int aio_fildes;
+            off_t aio_offset;
+            void *aio_buf;   // volatile
+            size_t aio_nbytes;
+            private int[2] __spare;
+            private void *_spare2__;
+            int aio_lio_opcode;
+            int aio_reqprio;
+            private __aiocb_private _aiocb_private;
+            sigevent aio_sigevent;
+        }
+
+        version = bsd_posix;
+    }
+    else version (NetBSD)
+    {
+        struct aiocb
+        {
+            off_t aio_offset;
+            void *aio_buf;   // volatile
+            size_t aio_nbytes;
+            int aio_fildes;
+            int aio_lio_opcode;
+            int aio_reqprio;
+            sigevent aio_sigevent;
+            private int _state;
+            private int _errno;
+            private ssize_t _retval;
+        }
+
+        version = bsd_posix;
```

Comment:
Not sure about this, does it really make things easier to have this instead of specific versions?

Don't mind either way.

Reply:
Given how small this module is, I mean.

Reply:
I think it focusses the attention on the difference between the enums ordering.

Would be happy to reverse it though, let's see if somebody would like it changed as well.

Reply:
Can we proceed with removing `version = bsd_posix`?  Just to speed up pulling this in, as this is the only part I'm hesitant about here.